packages/client/src/ActionsAndReducers/playerUi/helpers/handleStateOfWorldChanges.js

```diff
@@ -0,0 +1,17 @@
+import findAsset from '../../../Components/Mapping/helpers/findAsset'
+/** create a marker for the supplied set of details */
+export default (/* object */ payload, /* object */ allForces) => {
+  const detail = payload.detail
+  detail.data.forEach(force => {
+    force.assets.forEach(entry => {
+      // ok, get the asset
```

Comment:
Similar to a comment I made on a previous PR, it would be better if you could make the comments a little less informal and a bit more imperative. e.g.

// Loop through the assets object and return a new object with additional items.

Reply:
_will try harder_